pipelines/build/prTester/pr_test_pipeline.groovy

```diff
@@ -87,12 +87,14 @@ class PullRequestTestPipeline implements Serializable {
             
             String actualJavaVersion = updateRepo ? "jdk${javaVersion}u" : "jdk${javaVersion}"
             def excludedBuilds = ""
+            def activeNodeTimeout = ""
```

Comment:
Can be removed if we're removing if we're not specifying a node timeout in the regen jobs (along with passing it into the `regenerationScript` below)

Reply:
Yes.